log/sequencer.go

```diff
@@ -61,6 +61,8 @@ var (
 	// configuration should be changed instead.
 	// A factor <1 WILL lead to token shortages, therefore it'll be normalized to 1.
 	QuotaIncreaseFactor = 1.1
+
+	seqOpts = storage.NewGetOpts(storage.Sequence, false, trillian.TreeType_LOG)
```

Comment:
I think we could improve readability of this line and the like below. See my comment next to GetOpts type definition.